release-notes/aws/v9.3.0.md

```diff
@@ -0,0 +1,23 @@
+## :zap: Giant Swarm Release 9.3.0 for AWS :zap:
+
+This release includes Kubernetes v1.16.9 as well as reliability and user experience improvements.
+
+In addition, this release replaces CoreOS with Flatcar Container Linux.
```

Comment:
We now have a blog post from Timo about the reasons for the coreos -> flatcar move. Maybe we can link to https://www.giantswarm.io/blog/time-to-catch-a-new-train-flatcar-linux here.

Otherwise this looks good to me.

Reply:
See https://gigantic.slack.com/archives/CNUDS3AMU/p1588588887200400

Reply:
Sorry the slack link does not work. I thought we have the link already in Martas release notes.